### PR TITLE
[CI:DOCS] Add section about current differences

### PR DESCRIPTION
**Pull request by rsommer**
There are some differences in behaviour between docker and podman which should be documented to help developers while transitioning.

### CODE REVIEW

**Comment by mheon** [comment]
LGTM

**Comment by rhatdan** [comment]
Thanks @rsommer but you have to sign your commits.
git commit -a --amend -s
git push --force

Or simply add a line like

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

To you commit message.

LGTM
/approve

File: transfer.md [hunk 1]
```diff
@@ -89,6 +89,14 @@ There are other equivalents for these tools
 
 **** Use mount to take advantage of the entire linux tool chain rather then just cp.  Read [`here`](./docs/podman-cp.1.md) for more information.
 
+## Behavioural differences and pitfalls
+
+These commands behave different from Docker:
```
Comment (TomSweeneyRedHat) [inline transfer.md:94]:
```suggestion
These commands behave differently from the commands in Docker:
```

File: transfer.md [hunk 2]
```diff
@@ -89,6 +89,14 @@ There are other equivalents for these tools
...
+
+| Command | Description |
+| :--- | :--- |
+| `podman volume create` | While `docker volume create` is idempotent, `podman volume create` raises an error if the volume does already exist. See this [docker issue](https://github.com/moby/moby/issues/16068) on that.|
```
Comment (TomSweeneyRedHat) [inline transfer.md:98]:
```suggestion
| `podman volume create` | While `docker volume create` is idempotent, `podman volume create` raises an error if the volume does already exist. See this [docker issue](https://github.com/moby/moby/issues/16068) for more information.|
```

**Pull request merged**